Makefile

```diff
@@ -80,6 +82,11 @@ $(PANDA_KO): $(DRIVER_BUILD_DIR) $(DRIVER_BUILD_FILES) $(DRIVER_HEADER)
 driver: $(PANDA_KO)
 .PHONY: driver
 
+modules_install: $(DRIVER_BUILD_DIR)
+	CFLAGS_EXTRA=$(CFLAGS_EXTRA) $(MAKE) -C $(KERNEL_SRC) M="$<" modules_install \
+            ARCH=$(ARCH) CROSS_COMPILE=$(CROSS_COMPILE)
+
+.PHONY: modules_install
```

Comment:
Seems a bit inconsistent to be using `CROSS_COMPILE` here when it is really only an optional input parameter.  Should this be (implicitly?) using `CC` instead?

Reply:
Done!